ui/node/node.go

```diff
@@ -17,34 +17,39 @@ import (
 	"github.com/ory/x/stringslice"
 )
 
-// swagger:model uiNodeType
+// swagger:enum Type
```

Comment:
Could we name this

```suggestion
// swagger:enum UiNodeType
```

please?

Reply:
I would have to rename both the Swagger enum name and the actual Go type (`type UiNodeType string`), otherwise Swagger will not generate the spec correctly. Is that what you want?

Reply:
I see, yes that would be ok. The problem is that this will end up with `EnumType` in most generators, which is not unique enough and will cause issues when we have multiple enums called `type` - something that is quite likely to happen :)

Reply:
Done!